src/pagination/pagination.tsx

```diff
@@ -73,11 +80,16 @@ const Component = (
         isStart ? () => onHoverLeftButton() : () => onHoverRightButton();
     const blurAction = (isStart: boolean) =>
         isStart ? () => onBlurLeftButton() : () => onBlurRightButton();
-
+    const options: DropdownItemProps[] = [
+        { value: 10, label: "10 / page" },
+        { value: 20, label: "20 / page" },
+        { value: 30, label: "30 / page" },
+    ];
     // =============================================================================
     // EFFECTS
     // =============================================================================
     useEffect(() => {
+        setSelecctedOption(options[0]);
```

Comment:
init `selectedOption` directly with `useState`

could you add `activePage` to this `useEffect` dependency?